corehq/apps/domain/models.py

```diff
@@ -164,7 +166,16 @@ class InternalProperties(DocumentSchema, UpdatableSchema):
     goal_followup_rate = DecimalProperty()
     # intentionally different from and commtrack_enabled so that FMs can change
     commtrack_domain = BooleanProperty()
-    performance_threshold = IntegerProperty()
+    performance_threshold = IntegerProperty(default=15)
+    experienced_threshold = IntegerProperty(default=3)
```

Comment:
yes, sorry didn't think that one through enough thought it was something that didn't get saved often and would be easy to change. now i'm seeing this is part of the domain object...